kobo/apps/audit_log/models.py

```diff
@@ -354,3 +361,118 @@ def create_from_deployment_request(request):
             action=action,
             metadata=metadata,
         )
+
+    @classmethod
+    def create_from_detail_request(cls, request):
+        initial_data = getattr(request, 'initial_data', None)
+        updated_data = getattr(request, 'updated_data', None)
+
+        if initial_data is None or updated_data is None:
+            # Something went wrong with the request, don't try to create a log
+            return
+
+        asset_uid = request.resolver_match.kwargs['uid']
+        object_id = initial_data['id']
+
+        common_metadata = {
+            'asset_uid': asset_uid,
+            'log_subtype': PROJECT_HISTORY_LOG_PROJECT_SUBTYPE,
+            'ip_address': get_client_ip(request),
+            'source': get_human_readable_client_user_agent(request),
+        }
+
+        # always store the latest version uid
+        common_metadata.update(
+            {'latest_version_uid': updated_data['latest_version.uid']}
```

Comment:
why isn't this on the dictionary definition a few lines above? Looks like this update is unnecessary.